src/system/config.ts

```diff
@@ -8,7 +8,7 @@ import {
     Attribute,
     AttributeGroup,
     Resource,
-    Skill,
+    Skills,
```

Comment:
Is there any reason to make this plural? All the other config enums are singular and that is common naming practice for enums

Reply:
Because it's no longer an enum. It's now the list of skills as a constant.
Which in effect makes it like an enum, but means you can do more with how you access it. i.e. in this PR I had to use the string value passed in from a select box to select the right value, only as enum entries are treated kinda like Symbols they very much can't be swapped for their string equivalents. This way you can do that.

Then you use a little TS trick to get the types of the Keys as a union type which can act in places where you would have said "expect an entry from this enum", and *that* type is singular because it means this will be a singular instance of one of these constants.

Reply:
redundant now as the enum change is now reverted